contrib/format-maprdb/src/test/java/com/mapr/drill/maprdb/tests/json/BaseJsonTest.java

```diff
@@ -59,7 +59,7 @@ protected void runSQLAndVerifyCount(String sql, int expectedRowCount) throws Exc
   }
 
   private void printResultAndVerifyRowCount(List<QueryDataBatch> results, int expectedRowCount) throws SchemaChangeException {
```

Comment:
I guess we can rename this one to log as well.

Reply:
Fixed.